lib/compile.js

```diff
@@ -28,8 +30,8 @@ module.exports = function (filePath, options, done) {
       test.insertionIndex = test.attrs.flags.raw ?
         -1 : test.contents.indexOf(test262StreamMarker);
       test.contents = test.contents.replace(test262StreamMarker, '');
-
       test.file = path.relative(options.test262Dir, test.file);
+      test.source = source;
```

Comment:
Naming is hard and I'm afraid to create a bikeshedding, but I wonder if this should be exposed as `test.raw` as the raw source. It helps to distinguish between contents and raw without much further documentation. 

`raw` also follows the flags rule used in Test262.

In any case, the current code works for me.

Reply:
WFM (you can also push changes to my branch)

Reply:
Actually, I can/should/will take care of this.

Reply:
Not bad, thanks!